### PR TITLE
Add inview directive

### PR DESCRIPTION
**Pull request by Erbenos**
Addresses #25.

### CODE REVIEW

File: libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# inView directive
+
+Often times, one may be interested to know whether or not piece of DOM is visible on user's screen. There is [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) just for that purpose and this directive is very simple option-less wrapper around it.
```
Comment (fvoska) [inline libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md:3]:
```suggestion
Often times, one may be interested to know whether or not a piece of DOM is visible on the user's screen. We utilize [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) just for that purpose and this directive is a very simple option-less wrapper around it.
```

**Review by fvoska — CHANGES_REQUESTED** [review]

File: libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+`infInView` directive allows you to determine if a piece of DOM is visible on user's screen.
+
+## 2. Usage
+
+Simply add `infInView` directive to whatever DOM node you care about. You can then handle notifications from `(inView)` EventEmitter.
+
+```ts
+@Component({
+	selector: 'app-example',
+	template: `<div infInView (inView)="inView$.next($event)>{{ inView$ | async }}</div>`,
```
[the remark this answers is not in the document]
Comment (Erbenos) [reply libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md]: cb2a052

[the remark this answers is not in the document]
Comment (Erbenos) [reply libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md]: f33a7b5

File: libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md (hunk 1 above)
Comment (Erbenos) [reply libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md]: f33a7b5

File: libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# inView directive
+
+Often times, one may be interested to know whether or not piece of DOM is visible on user's screen. There is [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) just for that purpose and this directive is very simple option-less wrapper around it.
+
+## 1. Features
+
+`infInView` directive allows you to determine if a piece of DOM is visible on user's screen.
```
[the remark this answers is not in the document]
Comment (Erbenos) [reply libs/ngx-nuts-and-bolts/src/lib/directives/in-view/README.md:7]: f33a7b5

**Review by fvoska — APPROVED** [review]

**Pull request merged**